src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -196,7 +196,10 @@ public static bool IsPathRooted(string path)
         public static string GetPathRoot(string path)
         {
             if (path == null) return null;
-            return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
+			if (string.IsNullOrWhiteSpace(path))
```

Comment:
thanks, I always forget to check for that... I think VS is doing that and is ignoring the .editorconfig file at the root.